M/ModularForms/build_tarballs.jl

```diff
@@ -0,0 +1,38 @@
+using BinaryBuilder, Pkg
+import Pkg.Types: VersionSpec
+
+name = "ModularForms"
+version = v"0.1.0"
+
+sources = [GitSource("https://gitlab.com/mraum/ModularForms_jll_src.git",
+                     "4d81d41e625fde3e36dc86df5061313a8447cacd")]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/ModularForms_jll_src
+if [[ ${target} == *musl* ]]; then
+   export CFLAGS=-D_GNU_SOURCE
+fi
+./configure --prefix=$prefix --disable-static --enable-shared --with-gmp=$prefix --with-mpfr=$prefix --with-flint=$prefix --with-arb=$prefix CC=gcc ${extraflags}
+make -j${nproc} verbose
+make install LIBDIR=$(basename ${libdir})
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libmodularforms_jll", :libmodularforms_jll)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = [
+    Dependency(PackageSpec(name="Arb_jll"), compat = "~200.2200.0"),
+    Dependency(PackageSpec(name="FLINT_jll"), compat = "^200.800.401")
+]
+
+# Build the tarballs, and possibly a `build.jl` as well.
+build_tarballs(ARGS, name, version, sources, script, platforms, products, dependencies;
+               julia_compat="^1.6", preferred_gcc_version=v"11")
```

Comment:
So you said that compilation of a C code wouldn't succeed with versions of GCC before 11?

Reply:
It's the only one we work and test with. One needs C18, that's probably supported since earlier. I suppose this should then be as low as possible? I believe gcc9 already had full C18 support.

Reply:
For maximum compatibility, we should use the oldest version of GCC possible, yes.